financial-templates-lib/logger/PagerDutyTransport.js

```diff
@@ -28,7 +28,7 @@ module.exports = class PagerDutyTransport extends Transport {
           urgency: info.level == "warn" ? "low" : "high", // If level is warn then urgency is low. If level is error then urgency is high.
           body: {
             type: "incident_body",
-            details: info.mrkdwn ? info.mrkdwn : info // If the message has markdown then add it. Else put the whole info object.
+            details: info.mrkdwn ? info.mrkdwn : JSON.stringify(info) // If the message has markdown then add it. Else put the whole info object.
```

Comment:
Pagerduty patch. Now if there is `mrkdwn` in the info object it is printed. Else the whole `info` object is returned.